token/wallet.go

```diff
@@ -31,31 +31,31 @@ func WithType(tokenType string) ListTokensOption {
 
 // WalletManager defines the interface for managing wallets.
 type WalletManager struct {
-	ts api2.TokenManagerService
+	ts *ManagementService
```

Comment:
you chance to give `ts` a proper name

Reply:
just look at this:
```go
w := t.ts.tms.Wallet(identity)
```

t -> ts -> tms ..... where is the appendix I can lookup what is called here :P